robot/basestation/static/js/elements/controls/rover_controls.js

```diff
@@ -75,6 +75,8 @@ $(document).ready(function () {
       }
     });
 
+    console.log('twistMsg', twistMessage.linear.x)
+    console.log('twistMsg', twistMessage.angular.z)
```

Comment:
this shouldnt be there, even in master lol. I think its peter who pushed this directly to master lol